code/game/gamemodes/miniantags/hades/hades.dm

```diff
@@ -0,0 +1,223 @@
+/mob/living/simple_animal/hades
+	name = "hades"
+	real_name = "hades"
+	desc = "A strange being, clad in dark robes. Their very presence radiates an uneasy power."
+	speak_emote = list("preaches","announces","spits","conveys")
+	emote_hear = list("hums.","prays.")
+	response_help  = "kneels before"
+	response_disarm = "flails at"
+	response_harm   = "punches"
+	icon = 'icons/mob/EvilPope.dmi'
+	icon_state = "EvilPope"
+	icon_living = "EvilPope"
+	speed = 1
+	a_intent = "harm"
+	status_flags = CANPUSH
+	attack_sound = 'sound/magic/MAGIC_MISSILE.ogg'
+	death_sound = 'sound/magic/Teleport_diss.ogg'
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minbodytemp = 0
+	maxbodytemp = INFINITY
+	faction = list("hades")
+	attacktext = "strikes with an unholy rage at"
+	maxHealth = 500
+	health = 500
+	healable = 0
+	environment_smash = 2
+	melee_damage_lower = 75
+	melee_damage_upper = 85
+	see_in_dark = 8
+	see_invisible = SEE_INVISIBLE_MINIMUM
+	loot = list(/obj/effect/decal/cleanable/blood)
+	del_on_death = 1
+	deathmessage = "lets fourth a piercing howl, before sobbing quietly as they're dragged back to whence they came."
+
+	var/lastsinPerson = 0
+	var/sinPersonTime = 300
+	var/fakesinPersonChance = 60
+	var/list/sinPersonsayings = list("You revel in only your own greed.",\
+	"There is nothing but your absolution.",\
+	"Your choices have led you to this.",\
+	"There is only one way out.",\
+	"The only way to be free is to be free of yourself.",\
+	"Wallow in sin, and give yourself unto darkness.",\
+	"Only the truly sinful may stand.",\
+	"Find yourself and you will find Absolution.",\
+	"Forego the pain of this process, and submit.")
+
+/mob/living/simple_animal/hades/New()
+	..()
+	lastsinPerson = world.time
+	var/list/possible_titles = list("Pope","Bishop","Lord","Cardinal")
+	var/chosen = "[pick(possible_titles)] of Sin"
+	name = chosen
+	real_name = chosen
+
+	world << "<span class='warning'><font size=6>A [name] has entered your reality. Kneel before them.</font></span>"
+	world << 'sound/effects/pope_entry.ogg'
+
+	Appear()
+
+/mob/living/simple_animal/hades/hitby(atom/movable/AM, skipcatch, hitpush, blocked)
+	..(AM,skipcatch,hitpush,blocked)

```

Comment:
if you're not altering the arguments, just ```..()``` works fine, automatically passing on all args.